lib/github_changelog_generator/fetcher.rb

```diff
@@ -123,7 +124,11 @@ def fetch_closed_issues_and_pr
     def fetch_closed_pull_requests
       pull_requests = []
       begin
-        response = @github.pull_requests.list @options[:user], @options[:project], state: "closed"
+        if @options[:release_branch].nil?
+          response = @github.pull_requests.list @options[:user], @options[:project], state: "closed"
+        else
+          response = @github.pull_requests.list @options[:user], @options[:project], state: "closed", base: @options[:release_branch]
+        end
```

Comment:
In order to keep this method readable, can we make a method wrapping this GitHub API access (the whole `if..else..end`)?

What to call that method? I'm thinking of a name such as `_fetch_closed_pull_requests` in the worst naming-case. Perhaps something like `github_fetch_closed_prs`? Or `closed_prs_from_api`? What do you think?